.github/workflows/test-and-deploy.yml

```
@@ -95,7 +95,7 @@ jobs:
         docker: [bootstrap, core]
         project: [companion]
         new_project: [blueos]
-        platforms: ["linux/arm/v7"]
+        platforms: ["linux/arm/v7", "linux/arm64/v8"]
```

```suggestion
        platforms: ["linux/arm/v7,linux/arm64/v8"]
```

We'll need to run both in the same action, apparently. I'll do some reading on docs.